server/meca/file-generators/disclosure.test.js

```diff
@@ -1,12 +1,41 @@
+const PDFParser = require('pdf2json')
 const generatePdf = require('./disclosure')
 
 describe('Disclosure PDF generator', () => {
   it('returns a string of a PDF', async () => {
     const document = await generatePdf({
       meta: {},
-      teams: [{ role: 'author', teamMembers: [{ alias: {} }] }],
+      teams: [
+        {
+          role: 'author',
+          teamMembers: [
+            {
+              alias: {
+                givenNames: 'J. Edward',
```

Comment:
We could turn these values into variables so that the assertions can't become out of sync with the pdf values in future?

Reply:
@Cooryd I'm happy you making these changes